micro_mg3_0.F90

```diff
@@ -2105,7 +2138,18 @@ subroutine micro_mg_tend ( &
   if (do_sb_physics) then
      call sb2001v2_accre_cld_water_rain(qcic, ncic, qric, rho, relvar, pra, npra, mgncol*nlev)     
   else
-     call accrete_cloud_water_rain(microp_uniform, qric, qcic, ncic, relvar, accre_enhan, pra, npra, mgncol*nlev)
+
+     rtmp = qric 
+     ctmp = qcic 
+     ntmp = ncic 
+
+     if (accre_sees_auto) then
+       rtmp = rtmp + prc*deltat
+       ctmp = ctmp - prc*deltat
+       ntmp = ntmp - nprc*deltat    
+     endif 
```

Comment:
For GPU, it is better to add a do loop of mgncol and nlev, and write the calculations as `rmp(i,k) = qric(i,k)`.

Reply:
Done